### ISSUE TITLE
Settings should be editable using the GUI

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#206]
Right now all of Nammu's settings are stored in a YAML file in `~/.nammu/settings.yaml`. The intention is for users to be able to edit this file directly from the GUI.

This should include @stinney's suggestion to choose server for the validation/lemmatisation:

```
So, Nammu needs one of two things:
 
Either a) a way of configuring the build server statically
Or b) a drop-down menu to select the build server
 
The address of the Munich server is: oracc.ub.uni-muenchen.de
 
The address oracc.museum.upenn.edu will be reserved for web serving only, and there will be
a new address for the build server: could be boracc.museum.upenn.edu or 
build-oracc.museum.upenn.edu.
 
```

**Comment by raquel-ucl** [comment]
This issue can potentially get huge. I've prepared some wireframes on what I'd like to do, although I don't think all is functional for this milestone. Specially the buttons to add or remove existing languages/keystrokes/projects.

![general](https://cloud.githubusercontent.com/assets/10402246/22891099/2cccd3ca-f206-11e6-9cb3-ec4e76fb49b2.png)
![keystrokes](https://cloud.githubusercontent.com/assets/10402246/22891097/2cc984ea-f206-11e6-915f-c8ac994909f8.png)
![languages](https://cloud.githubusercontent.com/assets/10402246/22891098/2ccbe6fe-f206-11e6-937d-76301bdb3a8f.png)
![projects](https://cloud.githubusercontent.com/assets/10402246/22891096/2cc8e31e-f206-11e6-96a1-ad800671428f.png)

**Comment by EleanorRobson** [comment]
Agreed that this is potentially enormous! Almost all of it can wait. If I understand Steve correctly though, we'll need some sort of mechanism to specify the server in this release. He mentioned last Thursday that he'd be in touch with you. If he hasn't yet, shall I give him a nudge?

**Comment by raquel-ucl** [comment]
Hi @EleanorRobson, I thought I had replied to this, sorry. I haven't heard back from Steve, but it's OK because I'll send him the new release after our meeting today and he can try that one instead.

I have made the general settings tab for the working directory and to choose a server. The other tabs are a bit more complicated to polish, so they'll have to wait until the next release! All settings can be changed anyway directly on the settings file in case they need to.

See you later today!
